src/tests/jsonLogicFixtures.js

```diff
@@ -453,3 +435,132 @@ export const schemaWithPropertyThatDoesNotExistInThatLevelButDoesInFieldset = {
   },
   required: ['field_a'],
 };
+
+export const schemaWithInlineRuleForComputedAttributeWithCopy = {
+  properties: {
+    field_a: {
+      type: 'number',
+    },
+    field_b: {
+      type: 'number',
+      'x-jsf-logic-computedAttrs': {
+        title: {
+          value: 'I need this to work using the {{rule}}.',
+          rule: {
+            '+': [{ var: 'field_a' }, 10],
+          },
+        },
+      },
+    },
+  },
+};
+
+export const schemaWithInlineRuleForComputedAttributeWithoutCopy = {
+  properties: {
+    field_a: {
+      type: 'number',
+    },
+    field_b: {
+      type: 'number',
+      'x-jsf-logic-computedAttrs': {
+        title: {
+          rule: {
+            '+': [{ var: 'field_a' }, 10],
+          },
+        },
+      },
+    },
+  },
+};
+
+export const schemaWithInlineMultipleRulesForComputedAttributes = {
+  properties: {
+    field_a: {
+      type: 'number',
+    },
+    field_b: {
+      type: 'number',
+      'x-jsf-logic-computedAttrs': {
+        description: {
+          value: 'Must be between {{half_a}} and {{double_a}}.',
+          half_a: {
+            '/': [{ var: 'field_a' }, 2],
+          },
+          double_a: {
+            '*': [{ var: 'field_a' }, 2],
+          },
+        },
+      },
+    },
+  },
+};
+
+export const schemaSelfContainedValueForTitleWithNoTemplate = {
+  properties: {
+    field_a: {
+      type: 'number',
+    },
+    field_b: {
+      type: 'number',
+      'x-jsf-logic-computedAttrs': {
+        title: {
```

Comment:
thought (future iteration, not for now9: WDYT of a sugar syntax for this:

```
'x-jsf-logic-computedAttrs': {
  title: { '+': [{ var: 'field_a' }, 10] }
}
```

This can also lead us to computed min/max:

```
'x-jsf-logic-computedAttrs': {
  minimum: { '+': [{ var: 'field_a' }, 10] }
}
```

Which is an alternative to `x-jsf-logic-validations: ["bigger_than_a_plus_ten"]`

edit: I saw now the next example `schemaSelfContainedValueForMaximumMinimumValues`. Indeed it's an interesting case where we can add dynamic minimums in two ways. Either with `validations` or with `computations`. It's worth adding these to our guidelines — when to use one or another.

Reply:
No need for a sugar-syntax solution, this for MVP works perfectly. Regarding docs, do you already have a MR in place or just raw notes yet?

Reply:
I'm thinking the RFC, minus the research parts are going to be pretty useful as a start. Lets think about that in a bit.

Reply:
Agree that the RFC is a good starting point. 👍